### ISSUE TITLE
Process was not started

### ISSUE THREAD

**Issue by ekiyanov** [sosedoff/pgweb#160]
I have cloned sources, build it on target machine ( raspberri pi 2+ )
It was compiled, but when i'm trying to run it, it's literally doing nothing. 
`ps aux | grep pgweb ` also showing nothing. 

I tried in debug/release mode, with key -s ( to prevent browser opening ), providing connection data in different way , but nothing happens. 

non any single message or error...

**Comment by akarki15** [comment]
can you details how you went about running the binary? and what the output of invoking the binary was?

**Comment by ekiyanov** [comment]
There was no output at all. 

`./pgweb --bind=10.8.0.30 --listen=3005 -s --host=localhost -d --user=cambolive_user --db=cambolive_db`

`./pgweb --bind=10.8.0.30 --listen=8081 -s`

`./pgweb --url postgres://cambolive_user:secret_pass@cambolive_db:5432/postgres?sslmode=false --bind=10.8.0.30`

```
root@minibian:~/pgweb# ls -la
total 11136
drwxr-xr-x 11 root root     4096 Jun 12 10:52 .
drwx------ 13 root root     4096 Jun 14 03:08 ..
drwxr-xr-x  8 root root     4096 Jun 12 10:20 .git
-rw-r--r--  1 root root       17 Jun 12 10:20 .gitattributes
-rw-r--r--  1 root root       34 Jun 12 10:20 .gitignore
-rw-r--r--  1 root root      242 Jun 12 10:20 .travis.yml
-rw-r--r--  1 root root     6556 Jun 12 10:20 CHANGELOG.md
-rw-r--r--  1 root root      177 Jun 12 10:20 CONTRIBUTING.md
-rw-r--r--  1 root root      240 Jun 12 10:20 Dockerfile
drwxr-xr-x  2 root root     4096 Jun 12 10:20 Godeps
-rw-r--r--  1 root root     1108 Jun 12 10:20 LICENSE
-rw-r--r--  1 root root     2413 Jun 12 10:20 Makefile
-rw-r--r--  1 root root      107 Jun 12 10:20 Procfile.bak
-rw-r--r--  1 root root     3271 Jun 12 10:20 README.md
-rw-r--r--  1 root root      183 Jun 12 10:20 SCREENS.md
-rw-r--r--  1 root root      665 Jun 12 10:20 app.json
-rw-r--r--  1 root root      475 Jun 12 10:20 appveyor.yml
drwxr-xr-x  2 root root     4096 Jun 12 10:20 data
-rw-r--r--  1 root root     2512 Jun 12 10:20 main.go
-rwxr-xr-x  1 root root 11295392 Jun 12 10:32 pgweb
drwxr-xr-x 12 root root     4096 Jun 12 10:20 pkg
drwxr-xr-x  2 root root     4096 Jun 12 10:20 screenshots
drwxr-xr-x  2 root root     4096 Jun 12 10:20 script
drwxr-xr-x  5 root root     4096 Jun 12 10:26 src
drwxr-xr-x  6 root root     4096 Jun 12 10:20 static
drwxr-xr-x  4 root root     4096 Jun 12 10:20 vendor
```

**Comment by sosedoff** [comment]
@ekiyanov could you provide the following:

- go version used to compile sources
- build commands used to make a binary

RPI is not "officially" supported (no precompiled binaries) but i dont see why it wont run on the device. In fact i ran a cross-compiled binary on RPI once i think, didnt have any issues.

**Comment by sosedoff** [comment]
Pgweb now has support for ARMv5. Since i have a few raspberry pi's around i was able to verify that cross-compilation works fine. Im using Go 1.6 for all development just FYI.

I dont have any pre-built binaries out yet (will be available starting with 0.9.3) but you can build one yourself. Make sure to pull latest changes and then run `make release`. After its all done, grab a binary file ./bin/pgweb_linux_arm_v5` from the current directory and drop it onto your pi. Should work.

**Comment by sosedoff** [comment]
P.S Here's the command used to cross-compile: https://github.com/sosedoff/pgweb/blob/master/Makefile#L56-L58
